locations/spiders/mercator_sl.py

```diff
@@ -0,0 +1,17 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class MercatorSLSpider(SitemapSpider, StructuredDataSpider):
+    name = "mercator_sl"
+    item_attributes = {"brand": "Mercator", "brand_wikidata": "Q738412"}
+    allowed_domains = ["www.mercator.si"]
+    sitemap_urls = [
+        "https://www.mercator.si/sitemap.xml/sitemap/Store/1",
+        "https://www.mercator.si/sitemap.xml/sitemap/Store/2",
+    ]
```

Comment:
When they add `/3`, we'll be losing POIs.

Instead use all sitemaps with `/Store/` in them with:
```suggestion
    sitemap_urls = ["https://www.mercator.si/sitemap.xml"]
    sitemap_follow = ["/Store/"]
```

I'll do a PR because I'm gonna change the country code at the same time.

Reply:
Thanks. I wavered on it and ran out of time when prepping this to retest with changes

Reply:
FYI scrapy can cache requests, I use it basically all the time, but it sometimes causes issues when doing requests to the same url but with different headers.

`pipenv run scrapy crawl -s HTTPCACHE_ENABLED=True -s HTTPCACHE_EXPIRATION_SECS=604800`